changelog.txt

```diff
@@ -9,6 +9,7 @@
 * Fix - Updates to fraud protection.
 * Fix - Prevent Payment Request buttons from showing up in Composite Product pages.
 * Update - Updated @woocommerce/experimental package to v2.1.0.
+* Add - Add support for suggested gateway methods in WC-Admin
```

Comment:
Nit: may add the dot at the end of this entry? Similar to the `readme.txt` file.